cargo-espflash/src/main.rs

```diff
@@ -7,13 +7,7 @@ use std::{
 use cargo_metadata::{Message, MetadataCommand};
 use clap::{Args, CommandFactory, Parser, Subcommand};
 use espflash::{
-    cli::{
-        self, board_info, checksum_md5, completions, config::Config, connect, erase_flash,
-        erase_partitions, erase_region, flash_elf_image, make_flash_data, monitor::monitor,
-        partition_table, print_board_info, read_flash, save_elf_as_image, serial_monitor,
-        ChecksumMd5Args, CompletionsArgs, ConnectArgs, EraseFlashArgs, EraseRegionArgs,
-        EspflashProgress, FlashConfigArgs, MonitorArgs, PartitionTableArgs, ReadFlashArgs,
-    },
+    cli::{self, config::Config, monitor::monitor, *},
```

Comment:
Normally I'd like to avoid glob imports, but in this case I think it's justified